collection_requirements.rst

```diff
@@ -191,6 +191,19 @@ Also:
 See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
 
 
+List of requirements for collections to be included in Ansible
+==============================================================
+
+To be included in the `ansible` package, collections must meet the following criteria:
+
+* `development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html>`_
+* `Collection requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ (this document)
+* `Ansible documentation format <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html>`_ and the `style guide <https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#style-guide>`_
+* to pass the Ansible `sanity tests <https://docs.ansible.com/ansible/devel/dev_guide/testing_sanity.html#testing-sanity>`_
+* to have `unit <https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#testing-units>`_ and / or `integration tests <https://docs.ansible.com/ansible/devel/dev_guide/testing_integration.html#testing-integration>`_
+* Public bug tracker and accept Pull Requests.
```

Comment:
BTW, right now we include at least one collection which does not accept pull requests.

Reply:
Also, we have a discussion ticket for some of these points: https://github.com/ansible-collections/overview/issues/126

Reply:
I removed this point for a while